tests/security/stig/oscap_xccdf_eval_remote.pm

```diff
@@ -0,0 +1,47 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: Test "stig" hardening in the "scap-security-guide": detection mode with remote
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#93886, poo#104943
+
+use base 'stigtest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use bootloader_setup qw(add_grub_cmdline_settings);
+use power_action_utils "power_action";
+
+sub run {
+    my ($self) = @_;
+
+    add_grub_cmdline_settings('ignore_loglevel', update_grub => 1);
+    power_action('reboot', textmode => 1);
+    $self->wait_boot(textmode => 1);
+
+    select_console 'root-console';
+
+    # Get ds file and profile ID, ds file
+    my $f_ssg_sle_ds = $stigtest::f_ssg_sle_ds;
+    my $profile_ID = $stigtest::profile_ID;
+    my $f_stdout = $stigtest::f_stdout;
+    my $f_stderr = $stigtest::f_stderr;
+    my $f_report = $stigtest::f_report;
+
+    # Verify detection mode with remote
+    my $ret = script_run("oscap xccdf eval --profile $profile_ID --oval-results --fetch-remote-resources --report $f_report $f_ssg_sle_ds > $f_stdout 2> $f_stderr", timeout => 3000);
+    record_info("Return=$ret", "# oscap xccdf eval --fetch-remote-resources --profile $profile_ID\" returns: $ret");
+    if ($ret == 137) {
+        $self->result("fail");
```

Comment:
Given that we have bug opened, how about add soft_fail tag here? or adding the bug number?

Reply:
I keep "fail" and add bug number.